content/docs/components/training/chainer.md

```diff
@@ -1,8 +1,8 @@
 +++
 title = "Chainer Training"
-description = "See v0.7 docs for instrustions on using Chainer for training"
+description = "See v0.6 docs for instrustions on using Chainer for training"
```

Comment:
It's not clear what we mean by "v0.6 docs". We should specify that we mean **Kubeflow** v0.6 (as opposed to Chainer v0.6 or something else entirely). We should also include a link to the relevant v0.6 doc page. You can use the drop-down list on the Kubeflow docs to find v0.6, then navigate to the Chainer page to get the link for pasting into the docs.

Reply:
Done!